app/bundles/CoreBundle/Config/config.php

```diff
@@ -451,6 +452,14 @@
                 ],
             ],
 
+            'mautic.shortener' => [
+                'class'     => \Mautic\CoreBundle\Shortener\Shortener::class,
+                'arguments' => [
+                    'mautic.helper.core_parameters',
+                    'mautic.shortener.custom.api',
+                ],
+            ],
```

Comment:
Defining services are no longer necessary in M5 and I think we shouldn't add more but rather remove those that are defined in the config files.